trafilatura/metadata.py

```diff
@@ -117,7 +118,7 @@ def extract_meta_json(tree, metadata):
     for elem in tree.xpath('.//script[@type="application/ld+json" or @type="application/settings+json"]'):
         if not elem.text:
             continue
-        element_text = JSON_MINIFY.sub(r'\1', elem.text)
+        element_text = html.unescape(JSON_MINIFY.sub(r'\1', elem.text))
```

Comment:
How is html.unescape different from `string.encode().decode('unicode-escape')` here? We could maybe skip an import.

Reply:
`string.encode().decode('unicode-escape')` does not remove html entities like `&#13;` in my tests.

Reply:
That's correct, I'm thinking of modifying the `normalize_json()` function in `json_metadata` accordingly. That way we would could use it here as well, does that work for you?

Reply:
Yep. I added a test with html entities too.

Reply:
I just took care of it in 22e3075639e0d14a8ce34c87a9bff0d7d8ceab30. Please leave a message or a PR if you had something else in mind.